lib/trino-hdfs/src/main/java/io/trino/hdfs/TrinoFileSystemCache.java

```diff
@@ -429,6 +429,34 @@ public InputStream getWrappedStream()
         }
     }
 
+    private static class RemoteIteratorWrapper
+            implements RemoteIterator<LocatedFileStatus>
+    {
+        private final RemoteIterator<LocatedFileStatus> delegate;
+        @SuppressWarnings({"FieldCanBeLocal", "unused"})
+        private final FileSystem fileSystem;
+
+        public RemoteIteratorWrapper(RemoteIterator<LocatedFileStatus> delegate, FileSystem fileSystem)
+        {
+            this.delegate = delegate;
+            this.fileSystem = fileSystem;
```

Comment:
nit: `rnn`

Reply:
will pass on this one - this is only used internally and follows same code pattern as Input/Output stream wrappers.